lib/checks/navigation/region.js

```diff
@@ -26,7 +26,9 @@ function isLandmark (node) {
 	if (node.hasAttribute('role')) {
 		return landmarkRoles.includes(node.getAttribute('role').toLowerCase());
 	} else {
-		return implicitLandmarks.includes(node.nodeName.toUpperCase());
+		return implicitLandmarks.some((implicitSelector) => {
```

Comment:
That looks like magic to me. Can you add a comment about what's happening here?

Reply:
Looks like he's electing to use the `matchesSelector` utility to determine whether the node has an implicit landmark role, instead of simply checking whether the array of of implicit landmark roles includes the nodeName. I'm guessing this wasn't working for all nodes, since the nodeName of an element with an implicit landmark role is often different than the landmark role. E.g., given a `<header>` element, `#nodeName === 'HEADER'`, but its implicit role is 'banner'.

Reply:
This also allows the values in the role's "implicit" entry in the lookup table to be CSS selectors more complex than just the nodeName. E.g., `input[type="button"]`

Reply:
Thanks! I'd still love to see a comment in the code for future reading.